doc/src/quick_start_guide.md

```diff
@@ -31,9 +31,22 @@ fn main() {
 (If you already have a `build.rs` file, you should be able to just
 call `process_root` in addition to whatever else that file is doing.)
 
-That's it! Note that `process_root` simply uses the default settings.
-If you want to configure how LALRPOP executes, see the
-[advanced setup](advanced_setup.md) section.
+In this case, `process_root` simply uses the default settings, which takes
+files in `src/` ending with the `.lalrpop` extension, and generates
+corresponding Rust source files with the same name in `OUT_DIR`. If you want to
+configure how LALRPOP executes, see the [advanced setup](advanced_setup.md)
+section.
+
+Finally, you can add any of the generated files as modules in your Rust code
+using the [`lalrpop_mod!`][lalrpop_mod] macro. For example, if your grammar is
+called `grammar.lalrpop`, to use the generated `grammar.rs` as a Rust module
+simply called `grammar`, add the following:
+
+```rust
+lalrpop_mod!(grammar)
+```
```

Comment:
I found the part "include the generated code" to be slightly misleading, so here is a proposal for a more high-level (focused on what users _want to achieve_, and not how it works under the hood) and hopefully more direct-style wording. I also copy-paster the suggestion below because I think the two texts should be the same. Feel free to amend/reject if you don't think it's a good suggestion

```suggestion
The [`lalrpop_mod!`][lalrpop_mod] macro generates a wrapper module in your
crate so that you can use the generated parser from your code. For example,
if the source grammar is located in `grammar.lalrpop`, adding the following line
to `lib.rs` will create a corresponding `grammar` submodule (note that you can
also add this line to a `foo.rs` module definition instead, which will then
create a submodule `foo::grammar`):

```rust
lalrpop_mod!(grammar);
```
```